.github/workflows/nightly-testing.yaml

```diff
@@ -14,4 +14,8 @@ jobs:
   nightly-testing:
     name: Test Core on EKS
     uses: ./.github/workflows/test-eks.yaml
+    strategy:
+      matrix:
+        flavor: [upstream, registry1, unicorn]
+      max-parallel: 1 # Ensures that only one matrix job runs at a time
```

Comment:
Is this due to naming conflicts with resources? Wonder if we could add a prefix/suffix with the flavor to any resources that clash in [our tasks](https://github.com/defenseunicorns/uds-core/blob/matrix-nightly-build/tasks/iac.yaml#L19-L108)?

Downside of having everything sequential is PRs that update the EKS stack will take ~2 hours to complete all CI (including this one) 😬

Reply:
yeah the cluster is named the same thing for each of the flavors.